diff --git a/node_modules/next-contentlayer/dist/hooks/useMDXComponent.d.ts b/node_modules/next-contentlayer/dist/hooks/useMDXComponent.d.ts
index 1014387..32525e4 100644
--- a/node_modules/next-contentlayer/dist/hooks/useMDXComponent.d.ts
+++ b/node_modules/next-contentlayer/dist/hooks/useMDXComponent.d.ts
@@ -1,3 +1,3 @@
 import React from 'react';
-export declare const useMDXComponent: (code: string, globals?: Record<string, unknown>) => React.ComponentType<any>;
+export declare const useMDXComponent: (code: string, globals?: Record<string, unknown>) => { default: React.ComponentType<any> };
 //# sourceMappingURL=useMDXComponent.d.ts.map
diff --git a/node_modules/next-contentlayer/dist/hooks/useMDXComponent.js b/node_modules/next-contentlayer/dist/hooks/useMDXComponent.js
index 736478e..08928ef 100644
--- a/node_modules/next-contentlayer/dist/hooks/useMDXComponent.js
+++ b/node_modules/next-contentlayer/dist/hooks/useMDXComponent.js
@@ -15,7 +15,7 @@ import { _jsx_runtime } from './jsx-runtime.cjs';
 const getMDXComponent = (code, globals = {}) => {
     const scope = { React, ReactDOM, _jsx_runtime, ...globals };
     const fn = new Function(...Object.keys(scope), code);
-    return fn(...Object.values(scope)).default;
+    return fn(...Object.values(scope));
 };
 export const useMDXComponent = (code, globals = {}) => {
     return React.useMemo(() => getMDXComponent(code, globals), [code, globals]);
diff --git a/node_modules/next-contentlayer/src/hooks/useMDXComponent.ts b/node_modules/next-contentlayer/src/hooks/useMDXComponent.ts
index 83b0bcc..a2b1b50 100644
--- a/node_modules/next-contentlayer/src/hooks/useMDXComponent.ts
+++ b/node_modules/next-contentlayer/src/hooks/useMDXComponent.ts
@@ -16,7 +16,7 @@ import { _jsx_runtime } from './jsx-runtime.cjs'
 const getMDXComponent = (code: string, globals: Record<string, unknown> = {}): React.ComponentType<any> => {
   const scope = { React, ReactDOM, _jsx_runtime, ...globals }
   const fn = new Function(...Object.keys(scope), code)
-  return fn(...Object.values(scope)).default
+  return fn(...Object.values(scope))
 }
 
 export const useMDXComponent = (code: string, globals: Record<string, unknown> = {}): React.ComponentType<any> => {
